frontend/pages/index.tsx

```diff
@@ -48,228 +47,265 @@ type HomeProps = {
-              <div className={styles.cardInside}
-                   style={{backgroundImage: 'url("/images/bg_card.svg")'}}>
-                <div className="flex flex-col justify-center" style={{}}
-                >
-                  <div className="text-3xl font-medium">
-                    Share
+              <div className='text-center text-2xl my-4'>
+                {/* <div className="mb-4">For Researchers</div> */}
+                <div className="grid gridl-cols-1 sm:grid-cols-2 gap-8 pt-4">
+                  <div className="aspect-video grid place-items-center bg-center bg-cover group text-white bg-discover">
+                    <div className="group-hover:hidden text-4xl">Discover</div>
+                    <div className="hidden group-hover:block justify-center m-2">Discover software relevant to your research interest</div>
                   </div>
-                  <div className="text-lg mt-5">
-                    Showcase your research software and promote reuse by others.
+                  <div className="aspect-video grid place-items-center bg-center bg-cover group text-white bg-cite">
+                    <div className="group-hover:hidden text-4xl">Cite</div>
+                    <div className="hidden group-hover:block justify-center m-2">Version specific bibliography supports correct software citation</div>
                   </div>
                 </div>
               </div>
             </div>
-          </div>
-          <div className=""
-               data-aos="fade-up" data-aos-offset="0" data-aos-delay="100" data-aos-duration="400"
-               data-aos-easing="ease-in-out">
-            <div className={`${styles.card} h-full`}>
-              <div className={styles.cardInside}
-                   style={{backgroundImage: 'url("/images/bg_card.svg")'}}>
-                <div className="flex flex-col justify-center">
-                  <div className="text-3xl font-medium">
-                    Impact
-                  </div>
-                  <div className="text-lg mt-5">
-                    Enable developers and research organizations to monitor the impact of their
-                    research software.
-                  </div>
-                </div>
+
+            {/* <div className="text-2xl mt-8 mb-4">
+              The Research Software Directory is open source and jointly developed by
+              <div className="grid grid-cols-1 md:grid-cols-2 gap-8 place-items-center pt-8 text-black">
+                <a href="https://esciencecenter.nl" className="hover:cursor-pointer w-full grid place-items-center" target="_blank" rel="noreferrer">
+                  <LogoEscience width="100%"/>
+                </a>
+                <a href="https://hifis.net" className="hover:cursor-pointer w-full grid place-items-center" target="_blank" rel="noreferrer">
+                  <LogoHifis width="60%" />
+                </a>
               </div>
-            </div>
+            </div> */}
```

Comment:
Still needed?

Reply:
Yes, let's keep it for now, until we have the final front page.